src/examples/utilities/components.ts

```diff
@@ -319,22 +334,6 @@ const components = [
       }
     }
   },
-  {
-    name: 'retrieveComponentStyles',
-    props: {
-      id: {
-        required: true,
-        type: 'string',
-        description:
-          "Specifies the unique ID used to retrieve CSS styles from the theme's [components](/components/theme-object#components) object"
-      },
-      props: {
-        required: true,
-        type: 'Partial<ThemeProps<Partial<DefaultTheme>>>',
-        description: 'Provides component props which contain the context `theme` object'
-      }
-    }
-  },
```

Comment:
Since v9 still supports `retrieveComponentStyles`, should we mark as deprecated instead of removing the documentation?

Reply:
I considered that, but since it is rarely used, I really want to direct consumers to the right way as the "only" way :wink: